src/Symfony/Component/Intl/Countries.php

```diff
@@ -21,7 +21,7 @@
  */
 final class Countries extends ResourceBundle
 {
-    private static bool $optionalUserAssigned;
+    private static ?bool $allowUserAssigned = null;
```

Comment:
Maybe this is technically required for some reasons, but I'd prefer to not have nullable booleans. Can we make this option truly binary and give it a `false` default value?

Reply:
It should not be initialized by default. So that if it's used directly, an error will occur.

```suggestion
    private static bool $allowUserAssigned;
```

Reply:
when it's null, we populate it from the env var

Reply:
So, I found a way to implement all ideas from all three. 

- the internal property now is `bool` only, not nullable
- there is a public method to change this property at runtime if you wish to do so
- changed GHA a bit 🤏 so I can run the ENV tests in isolation (this can be done easier with PHPUnit 11)

Let me know. But I will have to pick up next Monday, as work calls now :)